core/invoker/src/main/scala/whisk/core/containerpool/docker/DockerContainerFactory.scala

```diff
@@ -67,13 +69,13 @@ class DockerContainerFactory(config: WhiskConfig,
       image = image,
       userProvidedImage = userProvidedImage,
       memory = memory,
-      cpuShares = config.invokerCoreShare.toInt,
+      cpuShares = cpuShares,
       environment = Map("__OW_API_HOST" -> config.wskApiHost),
-      network = containerArgs.network,
-      dnsServers = containerArgs.dnsServers,
+      network = containerArgsConfig.network,
+      dnsServers = containerArgsConfig.dnsServers,
       name = Some(name),
-      useRunc = config.invokerUseRunc,
-      parameters ++ containerArgs.extraArgs)
+      useRunc = dockerContainerFactoryConfig.useRunc,
+      parameters ++ containerArgsConfig.extraArgs.map { case (k, v) => ("--" + k, v) })
```

Comment:
`v` here is an an array of strings e.g. `[v1, v2]`per the comment above; is that what's intended here?

Reply:
yes